README.md

```diff
@@ -1,27 +1,65 @@
 # Jupyter Qt Console
 
+[![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/jupyter)
+[![Build Status](https://travis-ci.org/jupyter/qtconsole.svg?branch=master)](https://travis-ci.org/jupyter/qtconsole)
+[![Documentation Status](https://readthedocs.org/projects/qtconsole/badge/?version=stable)](http://qtconsole.readthedocs.org/en/stable/?badge=stable)
+
 A rich Qt-based console for working with Jupyter kernels,
 supporting rich media output, session export, and more.
 
-To install:
+This is a very lightweight widget that
+largely feels like a terminal, but provides a number of enhancements only
+possible in a GUI, such as inline figures, proper multiline editing with syntax
+highlighting, graphical calltips, and much more.
 
-    pip install qtconsole
 
-And run:
+![qtconsole](docs/source/_images/qtconsole.png)
 
-    jupyter qtconsole
+## Installation
 
-The qtconsole requires [PyQt][] or [PySide][],
-which cannot be installed with pip.
-These can be installed with [conda][]:
+### Prerequisites
+The qtconsole requires [PyQt](http://www.riverbankcomputing.com/software/pyqt/intro)
+or [PySide](http://pyside.github.io/docs/pyside).
+These requirements cannot be installed with pip.
+PyQt may be installed with [conda](http://conda.pydata.org/docs):
 
     conda install pyqt
 
-or your system package manager, e.g.
+or your system package manager, e.g. for Linux Debian/Ubuntu:
+
+    sudo apt-get install python3-pyqt5
+
+or:
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
```

Comment:
@takluyver I can update to the changes you mentioned in #86. I don't see much harm leaving in the corrected versions for PyQt 4. Though I am equally fine with removing. Do others have a preference?